src/pages/index.js

```diff
@@ -67,17 +160,32 @@ const cardSection = new Section(
   ".elements"
 );
 
-cardSection.renderItems();
+api.getInitialCards().then((cards) => {
+  cards.forEach((card) => {
+    renderCard(card);
+  });
+});
 
 // Functions
 function createCard(data) {
-  const card = new Card(data, "#card-template", handleImageClick);
+  const card = new Card(
+    data,
+    "#card-template",
+    handleImageClick,
+    handleDeleteCardForm,
+    likeButton
+  );
   return card.generateCard();
 }
 
 function renderCard(data) {
   const card = createCard(data);
   cardSection.addItem(card);
+  if (data.isLiked) {
```

Comment:
Needs correction: Please don't set likes like that, the card should initialize the likes state by itself